chipy_org/settings.py

```diff
@@ -334,8 +342,9 @@ def env_list(key, defaults=None, delimiter=","):
 NH3_ALLOWED_URL_SCHEMES = ["http", "https", "data"]
 NH3_STRIP_COMMENTS = True  # default, listed here for documentation
 
-RECAPTCHA_PUBLIC_KEY = env_var("NORECAPTCHA_SITE_KEY")
```

Comment:
Please remove the recaptcha changes.  These won't work in prod.  There are test keys we can add to your local .env file that are specific to local dev and always pass. I will follow up on this separately.